### DISCUSSION TITLE
Can mount existing save?

### DISCUSSION THREAD

**Discussion by sn0rlacks** [thijsvanloef/palworld-server-docker#174] [Q&A] [answered]
Problem:
I started a palworld save on my local machine. We've made some progress and would like to translate this save to a dedicated server. I've tried FTP'ing save files from my local machine to the /Pal folder in the container, but it doesn't seem to take any effect.

Proposed Solution:
Add functionality to support importing another save and having it be the save ran by the palworld server

**Comment by FurkanVG** [comment, marked as answer]
This is not an easy step. The files between a local coop save and dedicated server are built differently. See https://github.com/xNul/palworld-host-save-fix.
An integration will most likely not be included i think.

**Comment by peauc** [comment]
For what its worth, it worked flawlessly for me.
I moved a Windows save to my Linux server, I just had to configure the save folder name in the .ini file and it worked no need to edit file content.
EDIT: Host had an issue but palworld-host-save-fix I managed to do it.
